lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openstack-infra_manager-auth_key_pair.rb.rb

```diff
@@ -0,0 +1,4 @@
+module MiqAeMethodService
+  class MiqAeServiceManageIQ_Providers_Openstack_InfraManager_AuthKeyPair < MiqAeServiceAuthKeyPair
+  end
+end
```

Comment:
File name has a extra ".rb" in the filename (`miq_ae_service_manageiq-providers-openstack-infra_manager-auth_key_pair.rb.rb`) and this model derives from `AuthPrivateKey` so the parent class should be `MiqAeServiceAuthPrivateKey`